.github/workflows/pcw.yml

```diff
@@ -11,11 +11,11 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.8]
+        python-version: ['3.8', '3.10']
```

Comment:
I don't see any reason to keep 3.8 here , whole project already switched 3.10 . Even during testing of the code locally it is expected that people will use Docker_dev which also runs everything within python BCI container with python 3.10